apps/chat/src/components/Chat/ChatInput/ReplyVariables.tsx

```diff
@@ -0,0 +1,98 @@
+import { useCallback } from 'react';
+
+import { replaceDefaultValuesFromContent } from '@/src/utils/app/prompts';
+
+import { Prompt } from '@/src/types/prompt';
+
+import {
+  ConversationsActions,
+  ConversationsSelectors,
+} from '@/src/store/conversations/conversations.reducers';
+import { useAppDispatch, useAppSelector } from '@/src/store/hooks';
+
+import { PromptVariablesDialog } from './PromptVariablesDialog';
+
+export const ReplyVariables = () => {
```

Comment:
I think typo is here
```suggestion
export const ReplayVariables = () => {
```